README.md

```diff
@@ -1 +1,36 @@
-# Portfolio-setup-and-mobile-version-skeleton
\ No newline at end of file
+# Portfolio-setup-and-mobile-version-skeleton
+
+> This is the first milestone in the process of creating my portfolio website.
```

Comment:
- [ ] **Note:** According to Microverse guidelines commit messages should be **capitalized and imperative** as provided in this [link](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md). Right now, they are imperative but they are not capitalized. Going forward, kindly ensure that you capitalize your commit messages.

- [ ] **[OPTIONAL]** In your Pull Request summary, if you probably meant to have an unordered list, you should add a space after the **`-`** sign, like so:
```
- Created and index.html page
- Create a style.css page
```